bench.py

```diff
@@ -175,9 +169,9 @@ def parse_args(parser, size=None, loop_types=(),
                         help='Seed to pass as random_state')
     parser.add_argument('--dataset-name', type=str, default=None,
                         help='Dataset name')
-    parser.add_argument('--no-intel-optimized', default=False, action='store_true',
-                        help='Use no intel optimized version. '
-                             'Now avalible for scikit-learn benchmarks'),
+    _ = parser.add_argument('--no-intel-optimized', default=False, action='store_true',
```

Comment:
This is not what codefactor ci was saying
There was an extra comma at the line 180 (` 'Now avalible for scikit-learn benchmarks'),`)
And `_ = ...` code is unnecessary